app/controllers/relations_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RelationsController < ApplicationController
+  def index
+    @group = Group.find(params[:group_id])
+    @charges = @group.relations.includes(:charge).map(&:charge)
+  end
```

Comment:
- [ ] The expenses should be ordered by the most recent. Kindly implement that. If I have two orders, the first one should be the one I added last the last one should be the one I added first.

Reply:
@TedLivist Hi, done 👍🏼 .